.travis.yml

```diff
@@ -0,0 +1,79 @@
+stages:
+  - test
+  - commitlint
+  - name: release
+    if: branch = master AND type != pull_request
+
+sudo: required
+cache: bundler
+language: ruby
+
+services:
+  - docker
+
```

Comment:
Isn't this still required here?  I remember @daks mentioning that sometimes CI runs have failed without it.  Maybe he can reconfirm that.

```yaml
before_install:
  - bundle install
```

Reply:
Since Travis language is Ruby, `bundle install` is automatically called when running job. I've removed it in a bunch of formulas and so far everything works well.

Reply:
Yes, I'm aware of that. Maybe this has something to do with local testing and not just Travis. @daks,  any info?

Reply:
no really. Sometimes on travis, I needed to add the `gem install bundler` and `bundle install` directives, but maybe we were missing something that makes this unneeded.

Reply:
got the problem just now https://travis-ci.com/daks/apt-formula/jobs/199691936

Reply:
@n-rodriguez if you have an idea ^^

Reply:
Actually I'm not in favor of this because both `bundle install` commands don't have the same effects.
The one run by Travis is `bundle install --jobs=3 --retry=3 --path=${BUNDLE_PATH:-vendor/bundle}` which specify the path to installed gems directory.
The one you run doesn't include this option. It results in downloading dependencies twice because they're not installed in the same directory. (cache is ineffective)

Reply:
Also when you run `bundle exec kitchen test` I'm pretty sure it will use the gems in `${BUNDLE_PATH:-vendor/bundle}`

Reply:
> got the problem just now https://travis-ci.com/daks/apt-formula/jobs/199691936

`Could not locate Gemfile`

Reply:
Well.. tests pass : https://travis-ci.com/saltstack-formulas/keepalived-formula/jobs/199689231.

IMHO, adding `bundle install` is totally useless.

Reply:
@n-rodriguez Excellent points, please ignore this request.